analytics/analytics.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+
+window.trackingEnabled = () => {
+  var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+  var fx = navigator.userAgent.match(/Firefox\/(\d+)/);
+  var ie10 = navigator.userAgent.match(/MSIE 10/i);
+  var w8 = navigator.appVersion.match(/Windows NT 6.2/);
+  if (fx && Number(fx[1]) < 32) {
+    _dntStatus = 'Unspecified'; // bug 887703
+  } else if (ie10 && w8) {
+    _dntStatus = 'Unspecified';
+  } else {
+    _dntStatus = {'0': 'Disabled', '1': 'Enabled'}[_dntStatus] || 'Unspecified';
+  }
+
+  window.trackingEnabled = () => {
+    return _dntStatus != 'Enabled';
+  }
+  return window.trackingEnabled();
+};
```

Comment:
This construction is a bit hard to understand. Since we're doing this all inside an anonymous function anyway, can you not just unroll this? I also don't see the point in storing trackingEnabled on the window object if we're only going to use it here.